### PR TITLE
[doc] fixes in validator doc from discord comments

### PR DESCRIPTION
**Pull request by sblackshear**

### CODE REVIEW

**Comment by Clay-Mysten** [comment]
[@Alberto](https://mysten-labs.slack.com/team/U033TDEJ1U2) has already fixed this in:
https://github.com/MystenLabs/sui/pull/1994

**Comment by Clay-Mysten** [comment]
@sblackshear and @asonnino see https://mysten-labs.slack.com/archives/C02JTC4QJBU/p1653059720308319

**Review by huitseeker — DISMISSED** [review]
Superseded by #1994 but I'm happy to review if you want to rebase instead.

File: doc/src/learn/architecture/validators.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ Narwhal enables the parallel ordering of transactions into batches that are coll
...
 
-Now, once this is done Sui can execute all certificates that had their locks set, on one or multiple cores (currently). Obviously, transactions for earlier versions of objects need to be processed first (causally), and that reduces the degree of concurrency. Both the read and write set is determined by the transaction itself, and not dynamically based on the contents of the object at a specific version (not currently).
+Now, once this is done Sui can execute all certificates that had their locks set, on one or multiple cores (currently). Obviously, transactions for earlier versions of objects need to be processed first (causally), and that reduces the degree of concurrency. The read and write set of the transaction can be statically determined from its (object, version) inputs--execution can only read/write an object that was an input to the transaction, or that was created by the transaction.
```
Comment (huitseeker) [inline doc/src/learn/architecture/validators.md:49]: I'm mildly concerned about the tuple breaking the flow of plain english.
```suggestion
Now, once this is done Sui can execute all certificates that had their locks set, on one or multiple cores (currently). Obviously, transactions for earlier versions of objects need to be processed first (causally), and that reduces the degree of concurrency. The read and write set of the transaction can be statically determined from its versioned object inputs--execution can only read/write an object that was an input to the transaction, or that was created by the transaction.
```

**Review by huitseeker — APPROVED** [review]
👍

**Pull request merged**